app/uk/gov/hmrc/platopsapi/webhook/WebhookController.scala

```diff
@@ -86,6 +86,22 @@ class WebhookController @Inject()(
               else if (rsp2.header.status != 200) rsp2
               else                                Ok(details("Push request processed"))
             )
+          case Some(delete: GithubRequest.Delete) =>
```

Comment:
can we map event type to repo to make this clearer?
we probably only need to extract repo name and event type to make this decision